db/migrate/20180326193542_rename_switch_id_column_in_hardware.rb

```diff
@@ -0,0 +1,5 @@
+class RenameSwitchIdColumnInHardware < ActiveRecord::Migration[5.0]
+  def change
+    rename_column :hardwares, :switch_id, :physical_switch_id
```

Comment:
You should be fine with switch_id here since the table is still called switches the fk should be switch_id.  Physical_switches is just the name of the association

Reply:
Done